pkg/models/business_case.go

```diff
@@ -125,7 +125,6 @@ type BusinessCase struct {
 	LifecycleCostLines                  EstimatedLifecycleCosts `json:"lifecycleCostLines" db:"lifecycle_cost_lines"`
 	CreatedAt                           *time.Time              `json:"createdAt" db:"created_at"`
 	UpdatedAt                           *time.Time              `json:"updatedAt" db:"updated_at"`
-	SubmittedAt                         *time.Time              `json:"submittedAt" db:"submitted_at"`
```

Comment:
Is removing this from the models likely to cause issues when trying to insert new GRTFeedback? The column doesn't have a `NOT NULL` constraint, so maybe it's alright

Also - any thoughts on a migration to remove the column? I think it's ok either way, but may be something we want to ticket if we want to loop back to it later.

Reply:
Good questions. I'll test insertion and see if it causes errors. As for removing the column, let me run some GraphQL queries to see if there's any data there in the deployed environments.

Reply:
Wait, by GRT feedback, do you mean business cases? The only change I'm making to GRT Feedback is making a GraphQL field non-nullable.

Reply:
No error when creating business cases; there actually isn't a matching database column, migration V23 removed it.

Reply:
Ah - good catch on that migration!